tests/integration/helpers/app-version-test.js

```diff
@@ -1,25 +1,37 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
-import { shaRegExp, versionRegExp } from 'ember-cli-app-version/utils/regexp';
+import { shaRegExp, versionRegExp, dateRegExp } from 'ember-cli-app-version/utils/regexp';
 
 moduleForComponent('Integration | Helper | {{app-version}}', {
   integration: true
 });
 
 test('it displays only app version', function(assert) {
-  assert.expect(2);
+  assert.expect(3);
 
-  this.render(hbs`{{app-version hideSha=true}}`);
+  this.render(hbs`{{app-version hideSha=true hideDate=true}}`);
 
   assert.ok(this.$().text().match(versionRegExp), 'Displays version.');
   assert.ok(!this.$().text().match(shaRegExp), 'Does not display git sha.');
+  assert.ok(!this.$().text().match(dateRegExp), 'Does not display git date.');
```

Comment:
We should use `assert.notOk` here (please correct it above while you're are it 😀)